### PR TITLE
Bump versions of kurbo,cairo, pango deps

### PR DESCRIPTION
**Pull request by flxzt**
Simply bumping the versions seems to be enough, there are no API updates that would require any changes in piet this time (see changelog [here](https://github.com/gtk-rs/gtk-rs-core/releases/tag/0.20.0))

### CODE REVIEW

**Comment by xStrom** [comment]
As a first step we need to get the CI passing. I have done the required work to have `master` pass CI, which is in #573. It still requires review. I invited you as a collaborator so that you can review and approve that.

**Comment by xStrom** [comment]
Alright now with #573 merged this PR here needs to be rebased on `master`.

**Comment by flxzt** [comment]
Okay, I rebased on `master` and applied clippy's suggestions to replace a deprecated function from `kurbo`.
From my side this is now ready.

**Review by xStrom — APPROVED** [review]
Thanks!

We have a general policy that the PR author presses the merge button after approvals. This is because the author might want additional reviews or to make additional changes.

So go ahead and press the merge button when you're ready.

**Comment by flxzt** [comment]
Alright, thank you for making the effort to get this merged!

**Pull request merged**
